src/OpenSage.Game/Logic/Object/UpgradeDefinition.cs

```diff
@@ -4,16 +4,16 @@
 
 namespace OpenSage.Logic.Object
 {
-    public sealed class Upgrade : BaseAsset
+    public sealed class UpgradeDefinition : BaseAsset
```

Comment:
The C&C3 SDK calls this `UpgradeTemplate` - can we use that name too please?

Reply:
Will do after my current game session is finished